suite/tests/CMakeLists.txt

```diff
@@ -4134,6 +4134,11 @@ if (BUILD_CLIENTS)
       set(tool.drcacheoff.burst_traceopts_nodr ON)
       set(tool.drcacheoff.burst_traceopts_nopost ON)
       torunonly_drcacheoff(burst_traceopts tool.drcacheoff.burst_traceopts "" "" "")
+      set(tool.drcacheoff.windows-timestamps_nodr ON)
+      set(tool.drcacheoff.windows-timestamps_nopost ON)
+      torunonly_drcacheoff(windows-timestamps tool.drcacheoff.windows-timestamps "" "" "")
+      set(tool.drcacheoff.windows-timestamps-expectbase "offline-windows-timestamps")
+      set(tool.drcacheoff.windows-timestamps_timeout 600)
```

Comment:
This is quite long: 10 full minutes.  Does it need to be this high?  If we did end up with future failures causing timeouts we don't understand we'd rather have them as short as possible to avoid making the whole CI take multiple minutes longer.

Reply:
Agreed. That was while debugging the hang issue. Moved it to 240s.

Reply:
Do we need 240?  Most are 90.

Reply:
How about 120? I saw it timing out at 90 locally.

Reply:
Oh ok, if you have evidence it can take a while then higher is reasonable.  That is a long test though if it's >90s on a lightly loaded local machine.